test/index-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Sorry, I wasn't talking about `'use strict';` - That's fine. I was talking about the entire file change. Please revert to the tape implementation. =)